hypervisor/include/common/hypercall.h

```diff
@@ -194,7 +194,7 @@ int32_t hcall_pulse_irqline(struct vm *vm, uint16_t vmid, uint64_t param);
  *
  * @param vm Pointer to VM data structure
  * @param vmid ID of the VM
- * @irq_req: request command for IRQ set or clear
+ * @param ops request command for IRQ set or clear
```

Comment:
I was going to send such PR myself but then got suspicious about the definition. It does not make a lot of sense to me so I would like @yonghuah to comment here.

Reply:
Thanks Dave to fix the comment issue in hypercall.h.
@yonghuah will ACK it.
1. Since you have modified hypervisor source code, so we need to provide a Tracked-On field in your patch.
2. No need to add "Fixes #1353# in your patch which will automatically close the issue. Our QA team needs to verify bugs before closing it.

Reply:
> 2. No need to add "Fixes #1353# in your patch which will automatically close the issue. Our QA team needs to verify bugs before closing it.

@hwang37 , when would be a good time to update the https://projectacrn.github.io/latest/developer-guides/contribute_guidelines.html documentation? Is this mechanism using `Tracked-On` fully in place now? Thanks!

Reply:
> 2. No need to add "Fixes #1353# in your patch which will automatically close the issue. Our QA team needs to verify bugs before closing it.

Isn't that done **before** the patch gets merged?